src/renderer/components/ace-editor/ace-editor.tsx

```diff
@@ -169,11 +169,12 @@ export class AceEditor extends React.Component<Props, State> {
   }
 
   render() {
-    const { className, hidden } = this.props;
+    const { className, hidden, children } = this.props;
 
     return (
       <div className={cssNames("AceEditor", className, { hidden })}>
         <div className="editor" ref={e => this.elem = e}/>
+        {children}
```

Comment:
what's this? Was it missing?

Reply:
Not really, it is needed to pass the `<Spinner />` through.